ztag/transforms/mysql.py

```diff
@@ -18,17 +18,15 @@ def _transform_object(self, obj):
         if not results:
             return zout
 
-        if "protocol_version" in results:
-            zout.transformed["protocol_version"] = results["protocol_version"]
-        if "server_version" in results:
-            zout.transformed["server_version"] = results["server_version"]
-        if "capability_flags" in results:
-            zout.transformed["capability_flags"] = results["capability_flags"]
-        if "status_flags" in results:
-            zout.transformed["status_flags"] = results["status_flags"]
-        if "error_code" in results:
-            zout.transformed["error_code"] = results["error_code"]
-        if "error_message" in results:
-            zout.transformed["error_message"] = self.clean_banner(results["error_message"])
+        to_copy = ["protocol_version", "server_version", "capability_flags",
```

Comment:
**Personal preference** _([pull request commenting with a purpose](https://gist.github.com/andrewsardone/3fb9746e415d59784e97))_

this could be a set, but it doesn't matter 😉

Reply:
since the only purpose is to iterate over it sequentially (even though order doesn't matter), isn't a `list` more suitable?